frontend/components/RegionSelector.tsx

```diff
@@ -150,7 +150,7 @@ export function RegionSelector({
   );
 
   let current = selected?.current;
-  let generated = current?.date ?? mainInfo.generated;
+  let created = current?.date ?? mainInfo.created;
```

Comment:
```suggestion
  let generated = current?.date ?? mainInfo.generated ?? mainInfo.created;
```

Reply:
yea ok, if its a field that will be present in epimodel output eventually then that sounds ok